chromaprint.py

```diff
@@ -31,9 +32,19 @@ def _guess_lib_name():
     return ('libchromaprint.so.1', 'libchromaprint.so.0')
 
 
+def _load_library(name):
+    if sys.platform == 'win32':
+        try:
+            return ctypes.cdll.LoadLibrary(ctypes.util.find_library(name))
+        except TypeError:
+            raise OSError()
+    else:
+        return ctypes.cdll.LoadLibrary(name)
```

Comment:
Because we call `ctypes.cdll.LoadLibrary` on both branches, a simpler thing to do might be:

```py
try:
    name = ctypes.util.find_library(name)
except ...:
    ...
return return ctypes.cdll.LoadLibrary(name)
```